jormungandr/src/main.rs

```diff
@@ -1,3 +1,5 @@
+#![type_length_limit = "11401260"]
```

Comment:
can you document why you picked this value, why it is there?

Reply:
`// Rustc default type_length_limit is too low for complex futures`

still not enough. Can't we narrow it down to something sensible instead? Why do we need such a high value in first place?

Reply:
This one was suggested by compiler minus a zero in the end. The exact value doesn't matter, but it should give some headroom, so as futures get more complex, compiler keeps accepting them. I've rounded it to a less random looking one.